roles/matrix-bridge-appservice-slack/tasks/setup_install.yml

```diff
@@ -0,0 +1,66 @@
+---
+
+# If the matrix-synapse role is not used, `matrix_synapse_role_executed` won't exist.
```

Comment:
This kind of "Fail if matrix-synapse role already executed" check had better be moved to `init.yml`.
It's what we've done for the other bridges and it works better if it's there.